test/integration/setup.go

```diff
@@ -97,7 +113,12 @@ func setupBqOnly() (pb.MixerClient, error) {
 	if err != nil {
 		log.Fatalf("failed to create Bigquery client: %v", err)
 	}
-	metadata, err := server.NewMetadata(strings.TrimSpace(string(bqTableID)), btProject, "", schemaPath)
+	metadata, err := server.NewMetadata(
+		strings.TrimSpace(string(bqTableID)),
+		"google.com:datcom-store-dev",
```

Comment:
nit: use const literal instead of hardcoding?

Reply:
Done